radio/src/gui/128x64/model_outputs.cpp

```diff
@@ -401,30 +414,40 @@ void menuModelLimits(event_t event)
 #if defined(PPM_UNIT_US)
           lcdDrawNumber(LIMITS_OFFSET_POS, y, ((int32_t)ld->offset*128) / 25, PREC1|RIGHT);
 #else
-          if (abs(ld->offset) >= 1000) {
+          if (abs(ld->offset) == 1000) {
             lcdDrawNumber(LIMITS_OFFSET_POS, y, ld->offset/10, RIGHT);
           }
           else {
-            lcdDrawNumber(LIMITS_OFFSET_POS, y, ld->offset, PREC1|RIGHT);
+            GVAR_MENU_ITEM(LIMITS_OFFSET_POS, y, ld->offset, -1000, 1000, RIGHT|attr|PREC1, 0, event);
           }
 #endif
           break;
 
         case ITEM_OUTPUTS_MIN:
-          if (ld->min <= 0) {
+          if (ld->min == 0) {
```

Comment:
This was correct! Now display is broken with extended limits (has decimal removed only when limit is 100.0 exactly)